extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelContextRecorder.java

```diff
@@ -90,6 +93,8 @@ public void customize(RuntimeValue<CamelContext> context, RuntimeValue<CamelCont
         return new RuntimeValue((CamelContextCustomizer) context -> context.setShutdownStrategy(new NoShutdownStrategy()));
     }
 
```

Comment:
I've marked those recorded methods as deprecated. Maybe it's even ok to remove them ? I don't know what we have done in the past for that kind of situations.

Reply:
If it's for internal use only, I'd vote for removing them. 
If the user can use them, I'd say maybe deprecate and remove when introducing 3.14, the next LTS ?

Reply:
Agreed - I think for internal stuff in the past like recorder methods, we have removed them without deprecation. It's not public API.

Reply:
Okay I've removed the useless methods. Thanks all.